src/context/schemas/action.schema.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/action.schema.json",
+  "type": "object",
+  "title": "Action",
+  "properties": {
+    "type": { "const": "fdc3.action" },
+    "title": {
+      "type": "string"
+    },
+    "intent": {
+      "type": "string",
+      "$comment": "Should reference an intent type name, such as those defined in the FDC3 Standard'"
+    },
+    "context": {
```

Comment:
As per earlier comments an optional `app` element is needed with two string properties:
- (required) `appId`
- (optional) `instanceId`